pCrunch/runBatch/run_FlapGainSweep_BAR.py

```diff
@@ -5,9 +5,9 @@
 import numpy as np
 import yaml
 import os
-# WEIS tools
-from weis.aeroelasticse import runFAST_pywrapper, CaseGen_General
-from weis.aeroelasticse.Util import FileTools
+# WISDEM tools
```

Comment:
As long as there are some sort of examples, I think this can be scrapped. Especially because there is a control sweep example in `weis/examples/01_aeroelastics`